webflow-hacks/form-hacks/44/44.ts

```diff
@@ -0,0 +1,41 @@
+// when DOM is ready
+document.addEventListener('DOMContentLoaded', () => {
+  const FORM_SELECTOR = '[fs-hacks-element="form"]';
+  const SELECT_SELECTOR = '[fs-hacks-element="select"]';
+  const TOTAL_SELECTOR = '[fs-hacks-element="total"]';
+  const HIDDEN_TOTAL_SELECTOR = '[fs-hacks-element="hidden-total"]';
+  const form = document.querySelector(FORM_SELECTOR);
+  const totalDiv = document.querySelector<HTMLDivElement>(TOTAL_SELECTOR);
+  if (!form || !totalDiv) return;
+  const hiddenInput = form.querySelector<HTMLInputElement>(HIDDEN_TOTAL_SELECTOR);
+  if (!hiddenInput) return;
+  // get all selects
+  const selects = form.querySelectorAll<HTMLSelectElement>(SELECT_SELECTOR);
+  // listener for form
+  form.addEventListener('input', function (event) {
+    const target = event.target as HTMLInputElement;
+    // if target has attribute has the select selector
+    if (!target.matches(SELECT_SELECTOR)) return;
+    // loop through all selects
+    let total = 0;
+    selects.forEach((select: HTMLSelectElement) => {
+      total += Number(select.value);
+    });
```

Comment:
You should make sure that the values are not `NaN` after converting them to a number type, otherwise this code is prone to errors.